devtools/maven/src/main/java/io/quarkus/maven/RunMojo.java

```diff
@@ -0,0 +1,95 @@
+import io.quarkus.deployment.run.RunCommandHandler;
+import io.quarkus.deployment.run.RunCommandLaunchResultBuildItem;
+
+@Mojo(name = "run")
+public class RunMojo extends QuarkusBootstrapMojo {
+    /**
+     * The list of system properties defined for the plugin.
+     */
+    @Parameter
+    Map<String, String> systemProperties = Collections.emptyMap();
+
+    @Override
+    protected boolean beforeExecute() throws MojoExecutionException, MojoFailureException {
+        return true;
+    }
+
+    @Override
+    protected void doExecute() throws MojoExecutionException, MojoFailureException {
+        Set<String> propertiesToClear = new HashSet<>();
+
+        // Add the system properties of the plugin to the system properties
+        // if and only if they are not already set.
+        for (Map.Entry<String, String> entry : systemProperties.entrySet()) {
+            String key = entry.getKey();
+            if (System.getProperty(key) == null) {
+                System.setProperty(key, entry.getValue());
+                propertiesToClear.add(key);
+            }
+        }
+
+        try (CuratedApplication curatedApplication = bootstrapApplication()) {
+            AugmentAction action = curatedApplication.createAugmentor();
+            AtomicReference<Boolean> exists = new AtomicReference<>(false);
+            AtomicReference<String> tooMany = new AtomicReference<>();
+            action.performCustomBuild(RunCommandHandler.class.getName(), new Consumer<List>() {
+                @Override
+                public void accept(List cmd) {
+                    if (cmd.get(0) instanceof NullPointerException) {
+                        exists.set(false);
+                        return;
+                    } else {
+                        exists.set(true);
+                    }
+                    if (cmd.get(0) instanceof IndexOutOfBoundsException) {
+                        tooMany.set(((IndexOutOfBoundsException) cmd.get(0)).getMessage());
+                        return;
+                    }
+                    List<String> args = (List<String>) cmd.get(0);
+                    Path workingDirectory = (Path) cmd.get(1);
+                    try {
+                        Process process = new ProcessBuilder()
+                                .command(args)
+                                .inheritIO()
+                                .directory(workingDirectory.toFile())
+                                .start();
+                        int exit = process.waitFor();
+                    } catch (Exception e) {
+                        throw new RuntimeException(e);
+                    }
+                }
+            },
+                    RunCommandLaunchResultBuildItem.class.getName());
+            if (!exists.get()) {
+                getLog().info(
+                        "You do not need quarkus:run to execute your Quarkus project.  Just execute it like a plain Java application with java -jar");
```

Comment:
why not just have quarkus:run then run it (and print the command used)?

the jar path can be different dependent on version strings so `quarkus:run` just doing it makes it trivial rather than tedious.

Reply:
I implemented it this way because "some are saying" that they are afraid quarkus:run will be used for performance tests.

Can you not just do:
$ java -jar target/myapp-runn.jar

???  Why would there be any special command line switches?

Reply:
One option would be to support quarkus:run for everything, but have it bootstrap dev-services in the same way native tests do.

Reply:
Ok, I can add that too.

Reply:
Hmmm, although for lambda I wouldn't want to start the mock event server dev service. Mind if I hold off on that until I can implement quarkus:run for Lambda? I'd like to do that in a separate PR.

Reply:
How would you interact with it then if you don't have the mock event server? Either way this is not something that I think should hold up this PR.

Reply:
You can run Lambdas locally with 'sam local' command. (Azure Functions has something similar which this PR is implementing support for).